CHANGELOG.md

```diff
@@ -0,0 +1,109 @@
+# TRILLIAN Changelog
+
+## v1.0.6 - GetLeavesByRange. 403 Permission Errors. Signer Metrics.
+
+Published 2018-02-05 16:00:26 +0000 UTC
+
+A new log server RPC API has been added to get leaves in a range. This is a more natural fit for CT type applications as it more closely follows the CT HTTP API.
+
+The server now returns 403 for permission denied where it used to return 500 errors. This follows the behaviour of the C++ implementation.
+
+The log signer binary now reports metrics for the number it has signed and the number of errors that have occurred. This is intended to give more insight into the state of the queue and integration processing.
+
+Commit [b20b3109af7b68227c83c5d930271eaa4f0be771](https://api.github.com/repos/google/trillian/commits/b20b3109af7b68227c83c5d930271eaa4f0be771) Download [zip](https://api.github.com/repos/google/trillian/zipball/v1.0.6)
+
+## v1.0.5 - TLS, Merge Delay Metrics, Easier Admin Tests
+
+Published 2018-02-05 15:54:26 +0000 UTC
+
+The API protos have been rebuilt with GRPC 1.3.
+
+Timestamps have been added to the log leafs in the MySQL database. Before upgrading to this version you **must** make the following schema changes:
+
+Add the following column to the `LeafData` table. If you have existing data in the queue you might have to remove the NOT NULL clause:
+
+`QueueTimestampNanos  BIGINT NOT NULL`
+
+Add the following column to the `SequencedLeafData` table:
+
+`IntegrateTimestampNanos BIGINT NOT NULL`
+
+The above timestamps are used to export metrics via monitoring that give the merge delay for each tree that is in use. This is a good metric to use for alerting on.
+
+The Log and Map RPC servers now support TLS. 
+
+AdminServer tests have been improved.
+
+
+Commit [dec673baf984c3d22d7b314011d809258ec36821](https://api.github.com/repos/google/trillian/commits/dec673baf984c3d22d7b314011d809258ec36821) Download [zip](https://api.github.com/repos/google/trillian/zipball/v1.0.5)
+
+## v1.0.4 - Fix election issue. Large vendor updates.
+
+Published 2018-02-05 15:42:25 +0000 UTC
+
+An issue has been fixed where the master for a log could resign from the election while it was in the process of integrating a batch of leaves. We do not believe this could cause any issues with data integrity because of the versioned tree storage.
+
+This release includes a large number of vendor commits merged to catch up with etcd and GRPC v1.3.

```

Comment:
I think gRPC is the canonical capitalization (passim).

Do we know the etcd version that got vendored in?

Reply:
Done. Etcd was 3.2.0, added.